### ISSUE TITLE
DWG Review:  Create Base Shape Panel

### ISSUE THREAD

**Issue by sliptonic** [shaise/FreeCAD_SheetMetal#321]
Concept panel and tool

![baseshapepanel](https://github.com/shaise/FreeCAD_SheetMetal/assets/538057/70108355-6c90-4eb1-8c4b-17ff5a57131f)
![Baseshapetool](https://github.com/shaise/FreeCAD_SheetMetal/assets/538057/87dacf2b-b04a-4da3-b162-ae7e3bbf6cba)

**Comment by shaise** [comment]
This function already have a task panel. comments are appreciated:
![image](https://github.com/shaise/FreeCAD_SheetMetal/assets/5813568/e012d8ad-791e-43d7-9104-d5d14cb028cf)
note that there is no need for an apply button as every change in the fields of the panel are interactively shown on the part.

**Comment by maxwxyz** [comment]
My remarks: maybe add an option without a bend. Just to be able to create a flat sheet without the need for a sketch.
Also both tools seem to be called base shape (both issues have the exact same name @sliptonic) https://github.com/shaise/FreeCAD_SheetMetal/issues/328
Maybe this tool should keep the base shape name (or something like *default/standard presets*) and the other could be renamed to *Convert to Sheet* as it converts wires or closed sketches to a sheet.

**Comment by shaise** [comment]
Definitely agree for both remarks!

**Comment by maxwxyz** [comment]
Also, currently you create the feature and need the property editor to change parameters.
For all the issues and task dialogs, the idea is that you can double click on the feature in the tree and that same dialog opens and you can adjust the features in the task dialog and save changes.

**Comment by shaise** [comment]
Yes. This is already the way base shape is done now. Double clicking on it in the tree reopens it and let you make changes.

**Comment by shaise** [comment]
> My remarks: maybe add an option without a bend.

Added

**Comment by shaise** [comment]
Perhaps we need this dropdown to be images instead of text?

**Comment by maxwxyz** [comment]
I'm ok with the dropdown, they are descriptive and I think images would be too small on the dropdown scale for such complicated shaped (e.g. hat). Plus it's interactive so you can just change it and see the result.

**Comment by MisterMakerNL** [comment]
Ok not going to open a new ticket for this but would you be so kind to turn the grid of Qpushbuttons in a grid of Qradiobuttons, that would be waaaaaaaaaay better? IT causes issues with Minimum width of Qpushbuttons that was recommended to me, to set in the stylesheets.
![image](https://github.com/user-attachments/assets/6b9fef07-c21b-4abb-9dd6-f3cdf6c0f7a4)

**Comment by pierreporte** [comment]
@shaise Can we close this?

**Comment by shaise** [comment]
Closing
